src/style.css

```diff
@@ -1,3 +1,76 @@
+:root {
+  --header-bg-color: #fff;
```

Comment:
Using variables for all the colors (despite it's only required for blue) - nice!